src/platform/datapath_linux.c

```diff
@@ -2179,6 +2179,11 @@ QuicDataPathWorkerThread(
     QUIC_DATAPATH_PROC_CONTEXT* ProcContext = (QUIC_DATAPATH_PROC_CONTEXT*)Context;
     QUIC_DBG_ASSERT(ProcContext != NULL && ProcContext->Datapath != NULL);
 
+    QuicTraceEvent(
+        DatapathWorkerStart,
+        "[dpwrkr][%p] Start",
```

Comment:
Please change the `[dpwrkr]` to `[ udp]` and the string to `Worker start`. Same for the other events.

Reply:
Done